net.hhoney.tinycrate.yml

```diff
@@ -26,5 +26,5 @@ modules:
   build-commands:
   - install -Dm644 Tiny-Crate.pck ${FLATPAK_DEST}/bin/godot-runner.pck
   - install -Dm644 linux/${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
-  - install -Dm644 media/image/android_icon.png ${FLATPAK_DEST}/share/icons/hicolor/192x192/apps/${FLATPAK_ID}.png
+  - install -Dm644 media/image/android_icon.png ${FLATPAK_DEST}/share/icons/hicolor/128x128/apps/${FLATPAK_ID}.png
```

Comment:
the icon is truly 192x192, that's still incorrect.

Reply:
Fair that it should be 128x128 in an ideal world, though the docs are frustratingly unclear here. [Flathub](https://docs.flathub.org/docs/for-app-authors/linter/#appstream-missing-icon-file) says a 128px icon, [Flatpak](https://docs.flatpak.org/en/latest/conventions.html#application-icons) does not say a required size but <= 512px, and [FD.o](https://specifications.freedesktop.org/icon-theme-spec/latest/) doesn't say any certain sizes are valid or invalid afaict. :(

I guess I'll see if we can export a higher or lower size in the upstream repo.

Reply:
Upstream accepted a vector icon, so I've updated the manifest here.